openapi/kas-fleet-manager-private-admin.yaml

```diff
@@ -255,8 +255,6 @@ components:
               type: string
             subscription_id:
               type: string
-            sso_client_id:
```

Comment:
let's bump the version following this change https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/6e30f6b93614bfd64a4c1cb05f0a8298ba13bd70/openapi/kas-fleet-manager-private-admin.yaml#L5